SSA/Core/Tactic.lean

```diff
@@ -7,7 +7,7 @@ macro "simp_mlir": tactic =>
       (simp [TSSA.eval, Function.comp, id.def, TypedUserSemantics,
         TypedUserSemantics.eval, Context.Var,
         TypedUserSemantics.outUserType, TypedUserSemantics.argUserType,
-        UserType.mkPair, UserType.mkTriple] ; 
-       simp [bind_assoc, Option.bind_eq_some', Option.some_bind', 
-             Option.bind_eq_bind, pure])
+        UserType.mkPair, UserType.mkTriple,
```

Comment:
why do you remove the split here between the two `simp`s? Does this actually simplify more? (I would suspect it takes more time)

Reply:
Because the first simp sometimes resolves the goal, such that the second simp than triggers an error.